packages/amplify-e2e-tests/src/__tests__/with-babel-config.test.ts

```diff
@@ -0,0 +1,63 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import { spawnSync } from 'child_process';
+import {
+  initJSProjectWithProfile,
+  createNewProjectDir,
+  deleteProject,
+  deleteProjectDir,
+  nspawn as spawn,
+  getCLIPath,
+} from '@aws-amplify/amplify-e2e-core';
+
+describe('project with babel config', () => {
+  let projectRoot: string;
+  let packageJsonPath: string;
+  let babelConfigPath: string;
+  // eslint-disable-next-line spellcheck/spell-checker
+  const projName = 'withbabelconfig';
```

Comment:
Can you use camel case instead of disabling the spell checker? And then add `babel` to the dictionary (if not already there).